src/interaction/inspector.jl

```diff
@@ -742,20 +738,20 @@ function show_imagelike(inspector, plot, name, edge_based)
     return true
 end
 
-
 function _interpolated_getindex(xs, ys, img, mpos)
     x0, x1 = extrema(xs)
     y0, y1 = extrema(ys)
     x, y = clamp.(mpos, (x0, y0), (x1, y1))
 
     i = clamp((x - x0) / (x1 - x0) * size(img, 1) + 0.5, 1, size(img, 1))
     j = clamp((y - y0) / (y1 - y0) * size(img, 2) + 0.5, 1, size(img, 2))
-    l = clamp(floor(Int, i), 1, size(img, 1)-1);
-    r = clamp(l+1, 2, size(img, 1))
-    b = clamp(floor(Int, j), 1, size(img, 2)-1);
-    t = clamp(b+1, 2, size(img, 2))
-    z = ((r-i) * img[l, b] + (i-l) * img[r, b]) * (t-j) +
-        ((r-i) * img[l, t] + (i-l) * img[r, t]) * (j-b)
+    l = clamp(floor(Int, i), 1, size(img, 1) - 1)
+    r = clamp(l + 1, 2, size(img, 1))
+    b = clamp(floor(Int, j), 1, size(img, 2) - 1)
+    t = clamp(b + 1, 2, size(img, 2))
+    z =
+        ((r - i) * img[l, b] + (i - l) * img[r, b]) * (t - j) +
+        ((r - i) * img[l, t] + (i - l) * img[r, t]) * (j - b)
```

Comment:
I guess this is very subjective and nit-picky but I think in some situations (like here) it's nicer to have e.g. `(t-j)` instead of `(t - j)`. The former sort of establishes groups and gives you less separate things to look at which, in my opinion, helps readability. Maybe this is something to keep unformated?

Reply:
Not sure, I personally find this quite readable, and consistent with the rest.